content/modules/redisearch/_index.md

```diff
@@ -7,145 +7,67 @@ categories: ["Modules"]
+When combined with Redis Enterprise Software (RS), you can use the same RediSearch protocols and commands
+to get a geo-replicated, integrated query, and full-text search over efficient in-memory indexes.
+
+You can build compound indexes that index multiple fields in documents with text, numeric, and geospatial data types.
+With those indexes you can do language-aware fuzzy matching, fast auto-complete, exact phrase matching, numeric filtering, and geo-radius queries.
+
+As the documents in your database change, the index automatically processes these changes to keep your searches updated.
+You can also limit the index to only contain the index structure so that the results return a unique docID.
+
+For full-text searches, developers can customize the field queries and ranking of the search results.
+When querying, developers can use multiple predicates that query both text, numeric, and geospatial fields in one query.
+Sorting can be done by a given field as well and can be limited with an offset for simplified support for results pages.
+
+RediSearch is language aware and support the following languages for stemming:
+
+    "arabic", "danish", "dutch", "english", "finnish", "french", "german", "hungarian", "italian", "norwegian", "portuguese", "romanian", "russian", "spanish", "swedish", "tamil", "turkish"
+
+The index supports auto-complete engines with specific commands that can provide real-time interactive search suggestions.
 
 ## RediSearch Index
 
-The fundamental part of any search engine is what's called an Inverted
-Index. In very simple terms, a search index is a map between words or
-terms, to the respective documents they appear in.
-
-If we have two documents, one titled _Hello World_ and the other titled
-"Hello Kitty", a search engine would keep an Inverted Index of the word
-"hello" that contains two records, one for each of the aforementioned
-documents.
-
-Searching consists of loading, traversing, intersecting (if needed) and
-sorting these indexes in order to produce the most relevant results. It
-gets much more complicated, of course, as those indexes contain relevant
-scores, term positions in the document and more - but that's beyond the
-scope of this document.
-
-Until now, Inverted Indexes on top of a Redis database were always
-modeled with Redis native data types, usually Sorted Sets, where the
-document ID was stored as the "element" and the relevance as the
-"score." This worked great because it allowed developers to use Redis
-intersection and union to perform multi-word searches, but the approach
-has its limitations: it doesn't support exact phrase search, it has a
-big memory overhead, and can be very slow with big records
-intersections.
+The fundamental part of any search engine is the inverted index.
+The search index is a map between words or terms to the respective documents they appear in.
```

Comment:
An inverted index is mapping between terms (or words) and the documents they appear in.